source/testnet/guides/multisig-transfer.rst

```diff
@@ -0,0 +1,175 @@
+
+.. _create-multisig-scheduled:
+
+=====================================================================
+Create a scheduled multi-signature GTU transfer in the Desktop Wallet
+=====================================================================
+
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Multi-signature GTU transfer
+============================
+
+A multi-signature transfer of GTU is a transaction that
+requires two or more co-signers to sign with their private keys to
+authorize the transaction. You can schedule the transaction in advance so
+that a specific amount of GTU can be released on a specific date and at
+a specific time in the future.
+
+You can create two types of release schedules: a regular interval
+schedule and an explicit schedule.
+
+-  Use a regular interval schedule to release an equal amount of GTU to
+   a recipient at regular intervals.
+
+-  Use an explicit schedule if you want the intervals between releases
+   to be of different lengths, and if you want to be able to release
+   different amounts of GTU to the recipient at each interval.
+
+Prerequisites
+=============
+
+-  Set up an :ref:`identity and an account <create-initial-account-desktop>`.
+
+-  Set up a :ref:`Ledger Nano S hardware device identity <install-Ledger-app>`.
+
+Make a transfer proposal
+========================
+
+#.  Go to **Multi Signature Transactions**, and then select **Make new proposal**. All available transactions are listed in the right pane.
+
+#.  Select **Send GTU with a schedule**.
+
+#.  Select the relevant proposer. You’ll see a list of all the identities that you’re the custodian of. When you select an identity, the **Transactions Details** in the left pane are updated with the identity information and the current transaction expiry time. You'll be able to change the expiry time later in this process. Use the scrollbar if you can't see all the information. Select **Continue**.
+
+#. Select the relevant sender account from the list of available accounts, and then select **Continue**.
+
+#.  In **Amount**, enter the amount of GTU you want to transfer. If the amount you want to transfer including the transfer fee exceeds the amount in your account, you’ll see a message saying **Insufficient funds**. This means that there's not enough GTU in the account and you’ll have to change the amount. Also, the amount must be greater than zero. In the **Transaction details** under **Amount**, you can see the estimated fee. Select **Continue**.
+
+#.  Select the relevant recipient from the list. If there are many recipients in the list, you can use search to find the right recipient. Select **Continue**. You can now set up a release schedule.
+
+Set the **Transaction expiry** time. You must submit proposals to the chain within the last 2 hours up to the expiry date. This means you must set the expiry time so that it’s possible for any co-signers to return their signatures in time. Select **Continue**.
```

Comment:
Should we maybe highlight this a bit, so people don't end up missing the submission window?